tests/fixtures/direct_award_project_locked_fixture.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
It would be better not to add a new fixture for this but to mutate the existing fixture, e.g. by adding a `locked_at` parameter to the calling function to set the attribute to whatever the test needs. Might need to tweak implementation slightly for this, but it would be cleaner.